From: csagan5 <32685696+csagan5@users.noreply.github.com>
Date: Sat, 26 Mar 2022 16:41:55 +0100
Subject: Disable AsyncDNS by default

This feature is detrimental to privacy, see also:
* https://bugs.chromium.org/p/chromium/issues/detail?id=805020

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 .../net/base/features_cc/Disable-AsyncDNS-by-default.inc         | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 Spectre Browser_flags/net/base/features_cc/Disable-AsyncDNS-by-default.inc

diff --git a/Spectre Browser_flags/net/base/features_cc/Disable-AsyncDNS-by-default.inc b/Spectre Browser_flags/net/base/features_cc/Disable-AsyncDNS-by-default.inc
new file mode 100644
--- /dev/null
+++ b/Spectre Browser_flags/net/base/features_cc/Disable-AsyncDNS-by-default.inc
@@ -0,0 +1 @@
+SET_Spectre Browser_FEATURE_DISABLED(kAsyncDns);
--
